docs/guides/initialize-account.mdx

```diff
@@ -0,0 +1,191 @@
+---
+title: Initialize your Qiskit Runtime service account
+description: How to initialize your Qiskit Runtime account, with the option of saving credentials for easy use.
+
+---
+
+# Initialize your Qiskit Runtime service account
+
+Before using Qiskit Runtime, you have to initialize (instantiate) your account by submitting credentials.  These credentials can be manually submitted each time you initialize the Qiskit Runtime service, or you can save them for reuse.
+
+<Admonition type="note" title="Notes">
+      * If you are not using a trusted environment, follow the instructions in [Initialize the service in an untrusted environment](/docs/guides/cloud-setup-untrusted) instead. For example, if you are using a public comptuer.
+      * Follow [these instructions](/docs/guides/cloud-setup-rest-api) if you want to connect by using the REST API instead of using Qiskit.
+      * If necessary, use [this information](/docs/guides/quickstart-steps-org#firewall) to configure your firewall to enable access to the IBM Quantum API endpoints.
+</Admonition>
+
+## Before you begin
+
+Ensure that you have completed these steps:
+
+1. Ensure that you are a member of an IBM Cloud account.  See [Set up your IBM Cloud account](/docs/guides/cloud-setup) for instructions.
+2. Create (or have access to) at least one instance. Follow these steps to verify:
+    1. Log in to [IBM Quantum Platform](https://quantum.cloud.ibm.com).
+    1. Make sure that the correct account and region are selected in the account switcher in the header.
+    1. Go to the [IBM Quantum Platform dashboard.](https://quantum.cloud.ibm.com) If you have one or more instances shown, you're done with this step. Otherwise, [create an instance.](/docs/guides/instances#add-instance)
```

Comment:
The URLs in steps 1 and 3 are the same - we don't need to tell them to go to the dashboard because they should already be there.